lesson03/main.js

```diff
@@ -0,0 +1,17 @@
+const ul = document.getElementById("js-target");
+const imgSrcs = ["/img/bookmark.png","/img/messege.png"];
+
+for (let i=0; i < imgSrcs.length; i++){
+  const li = document.createElement("li");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+
+  a.href = `a${i}.html`;
+  a.textContent = `a${i}`;
```

Comment:
IMO
Sorry, I forgot about the numbers. If you want to number from 1 as Saki-san says, here it is.
```suggestion
  a.href = `a${i+1}.html`;
  a.textContent = `a${i+1}`;
```

Reply:
Thank you mai-san :）
I overlooked it.
58d8884dedb016e51b6cdfb17c55057383c5d685